projects/js-packages/components/components/icon-tooltip/index.tsx

```diff
@@ -57,7 +58,8 @@ const IconTooltip: React.FC< IconTooltipProps > = ( {
 		focusOnMount: 'container' as const,
 		onClose: hideTooltip,
 		className: 'icon-tooltip-container',
-		inline: true,
+		inline,
+		shift: true,
```

Comment:
Icon tooltip did not work well on smaller devices. So `shift` prop is necessary to let the popover make the necessary adjustments to the `placement` in order to make sure that it stays visible.

| Before | After |
|--------|--------|
| ![IMG_5266](https://github.com/Automattic/jetpack/assets/18226415/2b0d80f0-28b1-429d-9345-033328aef42d) | ![IMG_5267](https://github.com/Automattic/jetpack/assets/18226415/f898d97a-195d-45b6-85fd-fca1f8134539) |
| ![IMG_5268](https://github.com/Automattic/jetpack/assets/18226415/10908c9c-ec2a-4fe8-8218-9a514fa58a6f) | ![IMG_5264](https://github.com/Automattic/jetpack/assets/18226415/3010c4db-3012-4eb4-86ab-faba08de7b2d) |

Reply:
Do you think we should also pass in the `shift` property and default it to false as it was before? If it's a fix in every case then it should be okay like this, just asking to avoid unintentionally messing up the tooltip where others use it

Reply:
Yes, the component is broken everywhere on smaller devices without this prop.

Reply:
OK, I have reverted it to a default value of `false` as it was before because that seems to be a safe approach.